src/utils/beacon/beacon.tsx

```diff
@@ -30,6 +31,30 @@ export const useRemovePeer = () => {
       .then(refresh);
 };
 
+export const useRemovePeersByAccounts = () => {
+  const client = useQueryClient();
+  const getPeersForAccounts = useGetPeersForAccounts();
+
+  return async (pkhs: RawPkh[]) => {
+    const peersToRemove = getPeersForAccounts(pkhs);
+    const peersData = await client.fetchQuery({
+      queryKey: [PEERS_QUERY_KEY],
+      // getPeers actually returns ExtendedPeerInfo (with the senderId)
+      queryFn: () => WalletClient.getPeers() as Promise<ExtendedPeerInfo[]>,
```

Comment:
you don't need this client.fetchQuery at all here, this is more than enough
```typescript
const peers = await WalletClient.getPeers()
```

Reply:
Thank you, fixed